src/main/AndroidManifest.xml

```diff
@@ -14,6 +14,7 @@
         android:theme="@style/AppTheme"
         android:icon="@mipmap/ic_launcher"
         android:allowBackup="false"
+        android:usesCleartextTraffic="true"
```

Comment:
default value is already true

Reply:
Not for api 28+, that's why that is needed.